### PR TITLE
✅ Add test to useAuth hook #424

### LINKED ISSUE #424: Add test to check cookies in useAuth hook
We need check if the hook return a token when called

### PR DESCRIPTION
**Pull request by iurynogueira**
fix #424

### CODE REVIEW

File: frontend/src/hooks/useAuth.test.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { AUTH_COOKIE } from 'consts';
+import { useAuth } from '.';
+
+describe('useAuth hook test', () => {
```
Comment (edimossilva) [inline frontend/src/hooks/useAuth.test.tsx:4]: You can remove "test"

File: frontend/src/hooks/useAuth.test.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  it('should return the token when called', () => {
```
Comment (edimossilva) [inline frontend/src/hooks/useAuth.test.tsx:5]: Maybe use "when token exists"

And then, add another scenario considering "when token does not exist"?

**Review by edimossilva — COMMENTED** [review]
Nice! Give me coverage!

File: frontend/src/hooks/useAuth.test.tsx (hunk 2 above)
Comment (iurynogueira) [reply frontend/src/hooks/useAuth.test.tsx:5]: When the token not exist is returned undefined, i can coverage this case too.

Comment (iurynogueira) [reply frontend/src/hooks/useAuth.test.tsx:5]: Thanks @edimossilva, your testing teachings are improving this project even though you don't know `nadica de nada` anything about the frontend.

Comment (iurynogueira) [reply frontend/src/hooks/useAuth.test.tsx:5]: Added!

Comment (edimossilva) [reply frontend/src/hooks/useAuth.test.tsx:5]: Niceeeeee, happy to see that.

**Review by edimossilva — APPROVED** [review]

**Pull request merged**